fabric-game-rule-api-v1/src/main/java/net/fabricmc/fabric/mixin/gamerule/RuleKeyMixin.java

```diff
@@ -26,7 +27,7 @@
 
 @Mixin(GameRules.Key.class)
 public abstract class RuleKeyMixin implements RuleKeyExtensions {
-	/* @Nullable */
+	@Nullable
```

Comment:
am I the only one who likes to sort their annotations from shortest to largest? even intellij defaults to reverse order and it makes me iritat